src/main/java/dev/jbang/cli/Info.java

```diff
@@ -267,24 +295,13 @@ public Integer doCall() throws IOException {
 	}
 }
 
-@CommandLine.Command(name = "modulepath", description = "Prints module-path used for this application using operating system specific path separation.")
-class ModulePath extends BaseInfoCommand {
```

Comment:
was it your intention to remove modulepath?

Reply:
Yup, I added it when I added the module support without really understanding the difference between class-path and module-path in Java. I thought a module path would only contain modules and the class-path the rest of the dependencies that are not modules. But that's not the case.

To explain it a bit more: in Java choosing to use the class-path or the module-path is basically like a switch that tells Java to go into module mode or not. If you put everything on the class-path it will just work like old-fashioned unmodulerized Java. If you put things on the module-path it will switch to module mode and everything has to be modules or they will be automatically turned into modules.

In JBang that's now controlled by the `--module` parameter, if it exists we put the dependencies on the module-path, if not we use the class-path. But in both cases they will have the exact same list of dependencies. So it just didn't make sense to have a new command that returned the exact same information.